### PR TITLE
Feature/case view  - created frist draft of the Case View Page

### PR DESCRIPTION
**Pull request by Alisa1989**
In this pull request, I created a case view page.
This addition was necessary as it is specified in the project requirements.

The project is just getting started so there isn't much description about how the case view page should look/contain except for what can be understood from the [Trello Card](https://trello.com/c/Oa5WbNGJ/4-as-a-case-manager-i-can-review-information-about-the-familys-case-so-that-i-can-better-serve-them-and-reduce-the-time-period-of) I was working from.

In addition, I built this page not knowing the structure of the object that we will obtain with our API requests.

These were the required attributes:
- [x] Head of Household (main point of contact)
- [x] Name of Family Members
- [x] Family Size
- [x] Education Status (Parent(s) or Guardian)
- [x] SS Numbers (X out all but last 4 for now, use dummy data)
- [x] Prior History, Prior Episodes of Homelessness
- [x] Employment Status

Based on [this sample](https://www.figma.com/proto/iM4t3H148ndBvVf8WmkTuO/Family-Promise?node-id=171%3A3242&scaling=scale-down&page-id=0%3A1&starting-point-node-id=171%3A3242&show-proto-sidebar=1) and the [roadmap](https://bloomtech.notion.site/Family-Promise-CMS-Roadmap-2921b1e1a59342b8a0723f5fc6fe14ef) I also added:

- is active
- documents
- applications
- veteran status
- relation to the guardian
- birth date

https://user-images.githubusercontent.com/37967343/149222924-f2e05424-ed4a-42e1-bed8-79e69ea88a1a.mp4

### CODE REVIEW

**Review by EsdrasBrutus — APPROVED** [review]
Very nice work, your table is very clean and easy to read.

**Review by jake-linn — APPROVED** [review]
Great work! The video was easy to follow and the color scheme appears to be in line with what the stakeholder is looking for. Good job improvising mock endpoints the back end-end completes their ticket.

**Review by iam-saeed — APPROVED** [review]
Looks good! One suggestion is that I notice that you are prop drilling, and this could be simplified using Context API, or Redux. Also, as Esdras suggested you may want to get rid of the console.log's in your code before the merge.

**Comment by PollyYP** [comment]
Great work, Alex. The family table has every attributes that is needed. As the family attribute will grow more in the future, we can discuss how we will manage attribute objects as a prop using Redux later with the FE team.

File: src/components/pages/CaseView/FamilyTable.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+          relationshipToGuardian: "Child",
+          ssn: '123-45-678',
+          veteran: "No",
+          education: "High School",
+          employment: "Student",
+        },
+      ];
+      
+      function onChange(pagination, filters, sorter, extra) {
+        console.log('params', pagination, filters, sorter, extra);
```
[the remark this answers is not in the document]
Comment (Alisa1989) [reply src/components/pages/CaseView/FamilyTable.js:129]: good point

**Review by miketillman95 — COMMENTED** [review]
I like the page but as polly said we will probably move to context api/ redux so we can handle he state better

**Comment by PollyYP** [comment]
[As a case manager, I can review information about the family’s case, so that I can better serve them and reduce the time period of homelessness.](https://trello.com/c/Oa5WbNGJ/4-as-a-case-manager-i-can-review-information-about-the-familys-case-so-that-i-can-better-serve-them-and-reduce-the-time-period-of)

**Pull request merged**